CLAUDE.md

```diff
@@ -0,0 +1,153 @@
+# CLAUDE.md
```

Comment:
Hi! Could you move the unrelated changes to a separate PR? :-)

Reply:
Yeah.